### PR TITLE
Try run the echo-dialer on circle-ci

### PR DESCRIPTION
**Pull request by tomaka**

### CODE REVIEW

**Comment by tomaka** [comment]
Almost first try 🎉

**Comment by tomaka** [comment]
I also included a commlit that fixes the `echo` examples.
Right now these examples assume that messages are length-prefixed, but the JS code doesn't do that and just sends raw bytes instead.

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,12 +41,20 @@ jobs:
     docker:
       - image: rust
       - image: ipfs/go-ipfs
+      - image: node
+        command: ["bash", "-c", "git clone https://github.com/libp2p/js-libp2p &&
```
Comment (tomaka) [inline .circleci/config.yml]: @kirushik Just in case, do you think there is a better way to do this?

Comment (kirushik) [reply .circleci/config.yml]: Well, let's create a proper image with those steps baked in, maybe?
That way, we won't have to run the same build steps on every test run.

Where and how to set that up — that's another question. Let's have a talk in person tomorrow.

Comment (kirushik) [reply .circleci/config.yml]: https://hub.docker.com/r/parity/libp2p-js-echo-server/

**Comment by kirushik** [comment]
Damn, it's `parity/libp2p-js-echo-server:gitlab-ci`, and there's no `latest` tag

**Pull request merged**